stable/graylog/README.md

```diff
@@ -188,12 +194,14 @@ Note: The port list should be sorted by port number.
 
 
 ## Input TLS
-To enable TLS on input in Graylog, you need to specify the server private key and certificate. You can add them in `graylog.serverFiles` value. For example
+To enable TLS on input in Graylog, you need to specify the server private key and certificate. For example:
 
 ```yaml
 graylog:
+  tls:
+    enabled: true
```

Comment:
`graylog.tls.enabled: true` should not be shown as example in this section cause it does not requires. 
`graylog.tls` is only for Graylog Web UI. For the input it just need private key and certificate in `serverFiles`.

Reply:
Ah you had this as an input configuration for TCP/UDP, ok I'll put into a different section for configuring the web UI under TLS without a ssl proxy sidecar